railties/lib/rails/test_unit/reporter.rb

```diff
@@ -15,8 +15,12 @@ def aggregated_results # :nodoc:
       filtered_results.reject!(&:skipped?) unless options[:verbose]
       filtered_results.map do |result|
         location, line = result.method(result.name).source_location
-        "bin/rails test #{location}:#{line}"
+        "bin/rails test #{relative_path_for(location)}:#{line}"
       end.join "\n"
     end
+
+    def relative_path_for(file)
+      file.sub(/^#{Rails.root}\/?/, '')
```

Comment:
So the [comment still stands](https://github.com/rails/rails/pull/19571#discussion_r30532322), I just used `Rails.root` instead.